gtclang/test/atlas-integration-test/AtlasIntegrationTestCompareOutput.cpp

```diff
@@ -0,0 +1,196 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
```

Comment:
why is this file in the test folder and not in `src/dawn/integrationtest`?

Reply:
I'm not sure i understand. We cannot move to dawn right now because it depends on `gridtools/clang_dsl.hpp` which is in gtclang.

Reply:
wrong file and there was an atlas dependency